### ISSUE TITLE
Lag in the actual game cause the CV system to malfunction

### ISSUE THREAD

**Issue by GoldExplosion** [andreped/super-ml-pets#39]
Whenever the game lags, the CV system doesn't realize this and detects the Shop Slots again causing a disruption between the actual game and the engine. Proper detection of lags needed.

**Comment by GoldExplosion** [comment]
I would like to work on this. I also want to add proper error handling and other possible improvement to CV System. Can I take this issue?

**Comment by andreped** [comment]
Yeah, thats true. Right now we have a lot of hard coded solutions. It is nice for developing and stuff, but likely will lead to issues for other people. Great if this could be done in a better way.

Sure! Just go ahead :)

**Comment by andreped** [comment]
@GoldExplosion I was thinking of doing some improvements the framework. Actually trying to make a chrome extension such that people could try it with the web app in a more seemless manner. Mostly just to learn how to make web extensions.

Sadly, as the game has progressed quite a bit since last time I worked on this project, and both the `sapai` and `sapai-gym` projects not either been updated, the AI will likely not do well anymore, as the simulated and real environments have diverged.

**Comment by GoldExplosion** [comment]
That sounds interesting. I want to apologise for not closing this issue ( became busy with irl stuff). 

I want to contribute to this web extension if possible.
